docs/src/components/fields/multiselect.tsx

```diff
@@ -0,0 +1,32 @@
+'use client';
+
+import { FieldDemoFrame } from './frame';
+import { MultiselectFieldInput } from '../../../../packages/keystatic/src/form/fields/multiselect/ui';
+import { useState } from 'react';
+
+export const MultiselectFieldDemo = () => {
+  const [value, setValue] = useState<readonly string[]>([
+    'surfing',
+    'basketball',
+    'music',
```

Comment:
And `'chess'`?

Reply:
Or what about moving the `options` up into an `options` const and then using grabbing the type from that? Something like this:

```
const options = [
  { label: 'Surfing', value: 'surfing' },
  { label: 'Basketball', value: 'basketball' },
  { label: 'Music', value: 'music' },
  { label: 'Chess', value: 'chess' },
] as const;

type Options = (typeof options)[number]['value'];
```

Reply:
Ahh yeah this is cool, have applied to the multiselect and the select, was there any other place you thought the typing could be improved @ChrisLaneAU? 🙏🏻

Reply:
That looks good to me :)